ddtrace/contrib/trace_utils.py

```diff
@@ -179,55 +180,55 @@ def distributed_tracing_enabled(int_config, default=False):
     return default
 
 
-def int_service(pin, int_config, default=None):
+def int_service(pin=None, int_config=None, default=None):
+    # type: (Optional[Pin], Optional[IntegrationConfig], Optional[str]) -> Optional[str]
```

Comment:
Looking through the usages of `int_service` it looks like we really only have to deal with pin being None. I think integration config is always used and would always be required.

```suggestion
    # type: (Optional[Pin], IntegrationConfig, Optional[str]) -> Optional[str]
```

Reply:
Yes that's a good point but having a non-optional between two optional is a bit weird 🙁